src/mudlet.pro

```diff
@@ -271,23 +281,51 @@ unix:!macx {
     LUA_DEFAULT_DIR = $${DATADIR}/lua
 } else:win32 {
     MINGW_BASE_DIR_TEST = $$(MINGW_BASE_DIR)
-    isEmpty( MINGW_BASE_DIR_TEST ) {
-        MINGW_BASE_DIR_TEST = "C:\\Qt\\Tools\\mingw730_32"
+    contains( DEFINES, INCLUDE_MAIN_BUILD_SYSTEM ) {
+        # For CI builds or users/developers using the setup-windows-sdk.ps1 method:
+        isEmpty( MINGW_BASE_DIR_TEST ) {
+            MINGW_BASE_DIR_TEST = "C:\\Qt\\Tools\\mingw730_32"
+        }
+        LIBS +=  \
+            -L"$${MINGW_BASE_DIR_TEST}\\bin" \
+            -llua51 \
+            -llibhunspell-1.6
+
+        INCLUDEPATH += \
+             "C:\\Libraries\\boost_1_71_0" \
+             "$${MINGW_BASE_DIR_TEST}\\include" \
+             "$${MINGW_BASE_DIR_TEST}\\lib\\include"
+
+    } else {
+        # For users/developers building with MSYS2 on Windows:
+        isEmpty( MINGW_BASE_DIR_TEST ) {
+            # As error(...) terminates the build process immediately a
+            # multi-line message must be presented as a single string!
```

Comment:
We don't need an explanation of how the error() function works here.

Reply:
Well, it was the first one in that is more than one line - and the temptation to break the message up into several `error(...)` lines was there for me until I realised that that wouldn't work. Guess it can go though - lets see if anyone in the future falls into the trap though I guess they won't with this one to copy from - I must confess I wasn't aware of the `$$escape_expand(...)` function before this! :open_mouth:

Reply:
Done.